docs/PARAMETERS.md

```diff
@@ -112,6 +111,23 @@ Possible values are:
 Choose vcf to annotate.
 Different vcfs to be separated by commas.
 
+### --annotate_cache
+
```

Comment:
Could you clarify the edge cases, for example: using: `--annotate_cache` but not either `--snpEff_cache` or `--vep_cache`; using `--snpEff_cache` without `--annotate_cache`?

Reply:
won't work.
You need to use `--annotate_cache` with `--snpEff_cache` or/and `--vep_cache`
If you use only `--annotate_cache` it's not using cache
If you use only `--snpEff_cache` or/and `--vep_cache` it's not using cache either

Reply:
Yes, can you have that in the docs? It will not crash though, it will just not do what one thinks it does. Right?

Reply:
OK, I will specify all that then